projects/fourier/lib/fourier/services/generate/tuist.rb

```diff
@@ -14,9 +14,12 @@ def call
           dependencies = ["dependencies", "fetch"]
           Utilities::System.tuist(*dependencies)
 
-          generate = ["generate"]
-          generate << "--open" if open
-          Utilities::System.tuist(*generate)
+          cache_warm = ["cache", "warm", "--dependencies-only"]
```

Comment:
this assumes users would not want the dependencies in the project but that may not be always the case. What about creating a new fourier command `focus`? I think having both might be a better option here.

Reply:
I really can't think of a use case which is not weird enough to be useful only for a "navigated" tuist developer who knows he can do `tuist generate` if he wants everything, isn't it?

Reply:
We actually already have a `focus` command in `fourier`, so I'd just move that logic there.

Reply:
but I'd document the behavior and that we recommend people using `./fourier focus` instead of `./fourier generate`

Reply:
Make sense, I'll add that.

The fact that you and I didn't know `fourier focus` existed is exactly why I didn't want to add one more command :D

Reply:
✅